src/d3a/d3a_core/util.py

```diff
@@ -171,6 +171,10 @@ def make_ba_name(owner):
     return f"BA {owner.name}"
 
 
+def make_sa_name(owner):
```

Comment:
I suppose that SA stands for "Settlement Agent", right?
I would suggest moving this method inside the `SettlementAgent` class, as that's the only place where it is currently used.

Reply:
Not a good idea, especially for the future when we add more integration tests for the settlement market. Check out the usages of the make_ba_name and make_iaa_name, they are also used in the context of integration tests.

Reply:
Thanks @spyrostz. Your answer got me thinking so I ask here, even though the PR has been merged. I have a doubt: when should these integration tests fail? For example, suppose I change the implementation of `make_ba_name` to be something like:
```python
def make_ba_name(owner):
    return "Some wrong string or value"
```
Should end-to-end tests fail?
I personally think they should, because one "side" is expecting to receive `f"BA {owner.name}"`, but it's receiving something else. If this is the case, then it would be good to directly use the expected string in integration tests, without calling `make_ba_name()`. Using the existing implementation is tempting, because we wouldn't have to manually write the string; but it actually will not allow us to be alerted if the implementation is incorrect.

What is your opinion on this?